### ISSUE TITLE
Hi friend, Why are you not update now?

### ISSUE THREAD

**Issue by zclongpop123** [pyqt/maya2014-qt4#1]
I am waiting for this...

**Comment by mottosso** [comment]
Hi @zclongpop123, what is it that you would like me to update? Is the current release not working for you?

**Comment by zclongpop123** [comment]
I always use PyQt4 for maya, But it is too hard to build, I am a Chinese, too many times when I build it, it occur  errors, and I mental breakdown， so I follow you on github, do your fans, thanks...

**Comment by mottosso** [comment]
Ok, well you don't have to build it if you use this distribution.

What problems are you having?

**Comment by zclongpop123** [comment]
Yes, your maya2014-qt4 is working well on my windows and mac, very thanks. But your maya2015-qt4 repo is empty, so I am afraid when I use maya2015, I don't know how to build Qt for new version maya, my english a little bad, autodesk's doc is hard to me, and I can't find any video tutorial for this work, so I am wating for your update... thanks!

**Comment by mottosso** [comment]
Ah, sorry I have no plans to compile it for Maya 2015. :( But if you make or find a compiled distribution, I'd be happy to upload it here so that everyone can take part.

**Comment by zclongpop123** [comment]
Hi friend, When I build Qt for maya, sip is successed, but Qt always error, .\qtdirs.cpp(1) : fatal error C1083: 无法打开包括文件:“QCoreApplication”: No such file or directory.....

It mean's can't find QCoreApplication, I don't know how to fix it......

**Comment by mottosso** [comment]
Hi @zclongpop123,

Try following this part of the PyQt5 installation process.
https://github.com/pyqt/python-qt5/wiki/Updating-the-repository#compilation

It is for Qt 5, but should be identical to Qt 4. Have you installed Qt? It is typically located at `c:\Qt`.

**Comment by zclongpop123** [comment]
Oh thanks, I will try agin....

**Comment by zclongpop123** [comment]
an new error, error U1073  it sad don't know how to build webCore\debug\webcore.lib

**Comment by mottosso** [comment]
Sorry, I can't help you unless you provide more information. Could you post everything you typed, and all of the output?

Are you using mayapy to compile? It should look something like this.

``` bat
$ "c:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\amd64\vcvars64.bat"
$ set PATH=%PATH%;C:\Qt\4.8\msvc2013_64\bin
$ cd c:\users\zclongpop123\Downloads\PyQt-gpl-4.8\
$ C:\Program Files\Autodesk\Maya2015\bin\mayapy.exe configure.py
$ nmake
$ nmake install
```

**Comment by mottosso** [comment]
Actually, you'll need a version of Visual Studio that matches your Maya install, it needs to be the same version that Maya was compiled with, and it also needs to match the version of Qt.

For example, if you're running Maya 2013, then you'll need MSVC 2010 (I think, you'll have to look this up), and the Autodesk Modified version of Qt, available via their website.

**Comment by zclongpop123** [comment]
OK, I was follow Autodesk's doc step by step, now it success, but I can't push to your repo... here my address  https://github.com/zclongpop123/Maya-PyQt4

**Comment by mottosso** [comment]
That's great, @zclongpop123, happy to hear you got it working, and thanks for sharing.

I'll have a look a little later about getting into this repository, until then, it would be great if you could have a look at:
- forking the [Maya 2012 repository](https://github.com/pyqt/maya2012-qt4)
- clone it to your local disk
- put the compiled PyQt4 for Maya 2012 in it
- push to your fork
- submit a pull-request

And then do the same for:
- Maya 2013
- Maya 2014
- Maya 2015

It's quite a bit of work, I'd understand if you think it's too much, but thanks for trying anyway.

**Comment by zclongpop123** [comment]
Now I forked your repo, and publish it. Thank you for help me....
